Add random subsampling as alternative to filtering (#39)
qbic-pipelines/vcftomat: docs/usage.md

> [!NOTE]
> The filtering step only works with conda for nextflow versions above 24.10.2 (use docker or singularity if you want to use an older nextflow version)

> ### Subsample VCFs

Reviewer: If you change the heading in `docs/output.md` then please change it here as well.

> VCF files can be randomly subsetted to keep only a specific fraction of variants. This enables comparison to the filtered variants.

Reviewer: ```suggestion
VCF files can be randomly selected to keep only a specific fraction of variants. This enables comparison to the filtered variants.
```

> You can determine appropriate fractions by comparing the number of filtered variants with the total number of variants. This can be done with a script that collects the number of variants by using `bcftools stats` from both files and dividing them. The more VCF files you use for comparison, the more robust the fraction becomes. (We compared around 90 files and obtained an average fraction of 0.00175 when using `--filter 'INFO/CSQ ~ "HIGH"'`).

Reviewer: ```suggestion
You can determine appropriate fractions by comparing the number of filtered variants with the total number of variants. This is done using a script that collects the number of variants with `bcftools stats` from both files and divides the number of variants before and after filtering. The more VCF files are used for comparison, the more robust the fraction becomes. (We compared around 90 files and obtained an average fraction of 0.00175 when using `--filter 'INFO/CSQ ~ "HIGH"'`).
```

> You can determine appropriate fractions by comparing the number of filtered variants with the total number of variants. This can be done with a script that collects the number of variants by using `bcftools stats` from both files and dividing them. The more VCF files you use to compare the more robust the fraction becomes. (We compared f.ex. around 90 files and landed at an average fraction of 0.00175 when using `--filter 'INFO/CSQ ~ "HIGH"'`).

Reviewer: ```suggestion
You can determine appropriate fractions by comparing the number of filtered variants with the total number of variants. This can be done with a script that collects the number of variants by using `bcftools stats` from both files and dividing them. The more VCF files you use for comparison, the more robust the fraction becomes. (We compared around 90 files and obtained an average fraction of 0.00175 when using `--filter 'INFO/CSQ ~ "HIGH"'`).
```
